eng/pipelines/runtime-ioslike-coreclr.yml

```diff
@@ -0,0 +1,21 @@
+# This is a wrapper yml for `runtime-extra-platforms-ioslike.yml`, which
+# has all the iOS/tvOS jobs. This file is essentially so we can have point
+# the pipeline in azdo UI to this, and thus avoid any scheduled triggers
```

Comment:
Nit: I think wording can be improved here, as it is a bit unclear what the intention is.